### PR TITLE
DM-33974: Create a "Butler worker" VM on Google

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR adds instructions to the Playbook for using the new VM, in particular for working with Google Storage repositories.

### CODE REVIEW

File: doc/playbook.rst [hunk 1]
```diff
@@ -195,3 +207,73 @@ On a VM with the Science Pipelines installed, a new APDB schema can be created i
...
+
+Built-in support:
+
+* a complete install of the Science Pipelines in ``/software/lsst_stack/``
+* a running instance of ``cloud_sql_proxy`` mapping the ``butler-registry`` database to port 5432
+* global configuration pointing Butler ``s3://`` URIs to Google Storage buckets
+
+The user is responsible for:
+
+* running ``/software/lsst_stack/loadLSST.sh`` on login
```
Comment (ebellm) [inline doc/playbook.rst:227]:
```suggestion
* running ``source /software/lsst_stack/loadLSST.sh`` on login
```

File: doc/playbook.rst [hunk 2]
```diff
@@ -195,3 +207,73 @@ On a VM with the Science Pipelines installed, a new APDB schema can be created i
...
+    registry:
+      db: postgresql://postgres@localhost:5432/
+      namespace: <unique namespace>
+
+Then run ``butler create --seed-config seedconfig.yaml <repo location>`` to create the repository.
+
+Each repository needs its own ``namespace`` value, corresponding to a PostgreSQL schema.
+Schemas can be listed from within ``psql`` using the ``\dn`` command, and corrupted or outdated registries can be deleted using the ``DROP SCHEMA`` command.
+
+.. warning::
```
Comment (ebellm) [inline doc/playbook.rst:262]: (I'm not certain what markup you're trying to use here but it doesn't seem to have rendered in the output?)

File: doc/playbook.rst [hunk 3]
```diff
@@ -195,3 +207,73 @@ On a VM with the Science Pipelines installed, a new APDB schema can be created i
...
+----------------------------------
+
+All Google Storage repositories must also be `PostgreSQL-Backed Repositories`_.
+Otherwise, no special configuration is needed to create one.
+
+To create or access a Google Storage repository, give the repository location as a URI, e.g.::
+
+    butler query-collections s3://<bucket name>/<repo location in bucket>
+
+.. important::
```
Comment (ebellm) [inline doc/playbook.rst:278]: Same markup question here

File: doc/playbook.rst [hunk 4]
```diff
@@ -195,3 +207,73 @@ On a VM with the Science Pipelines installed, a new APDB schema can be created i
...
+* running ``/software/lsst_stack/loadLSST.sh`` on login
+* database authentication (see `Databases`_, above)
+* `Google Storage Authentication`_
+
+
+Google Storage Authentication
+-----------------------------
+
+To access `Google Storage-Backed Repositories`_, you must first set up Boto authentication.
+If you don't have one, `create an HMAC key`_ (this is *not* the same as the token for running the `tester`_); the relevant service account is ``service-620570835826@gs-project-accounts.iam.gserviceaccount.com``.
```
Comment (ebellm) [inline doc/playbook.rst:236]: I was able to use successfully the `prompt-image-upload@prompt-proto.iam.gserviceaccount.com`--not sure if that matters

**Review by ebellm — APPROVED** [review]

File: doc/playbook.rst (hunk 2 above)
Comment (kfindeisen) [reply doc/playbook.rst:262]: It's a [standard directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions); it's unfortunate that GitHub doesn't support it, but it's better than nothing.

File: doc/playbook.rst (hunk 4 above)
Comment (kfindeisen) [reply doc/playbook.rst:236]: I think it matters a little bit from a tidiness perspective; given that it's what we were given for GS access (e.g., through `gsutil`), I see `service-620570835826` as an account for humans, and `prompt-image-upload` as reserved for `upload.py`.

**Pull request merged**